pyfolio/capacity.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import division
+import pandas as pd
+import numpy as np
+from . import pos
+from . import timeseries
+
+DISPLAY_UNIT = 1000000
+
+
+def get_assets_dollar_volume(positions, market_data,
```

Comment:
Would it make things more readable if we ommited `dollar` and only called it volume (we can't be sure it's dollars to begin with). Then, `DV` could become `vol` which is also more readable.